packages/devui/src/Tabs/Tabs.tsx

```diff
@@ -67,6 +61,8 @@ export default class Tabs<P> extends Component<TabsProps<P>> {
   }
 
   render() {
+    this.updateTabs(nextProps);
```

Comment:
`nextProps` isn't defined here which is why the build is failing. Can you instead turn this into a `renderTabs` and remove the `tabsHeader` class variable altogether? Thanks!

Reply:
Or you can just pass in `this.props` for now if that's easier.